src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -203,6 +203,7 @@ static void on_read(void* arg, grpc_error_handle err) {
     if (fd < 0) {
       switch (errno) {
         case EINTR:
+        case ECONNABORTED:
```

Comment:
Since this is a Linux behavior that's different from BSD should we consider translating the result inside grpc_accept4?

Reply:
translate it to what?

Reply:
I'm gonna go ahead and merge this since I know you are OOO at the moment. Any suggested change can be made later in a subsequent PR.